synapse/lib/httpapi.py

```diff
@@ -464,6 +464,33 @@ class AuthUserV1(Handler):
 
         self.sendRestRetn(user.pack())
 
+class AuthUserPasswdV1(Handler):
+
+    async def post(self, iden):
+
+        if not await self.reqAuthUser():
+            return
+        current_user = await self.user()
+
+        body = self.getJsonBody()
+        if body is None:
+            return
+
+        user = self.cell.auth.user(iden)
+        if user is None:
+            self.sendRestErr('NoSuchUser', f'User does not exist: {iden}')
+            return
+
+        password = body.get('password')
```

Comment:
In all other serialized forms ( such as telepath ) this field would be named `passwd`.  Lets stay consistent.

Reply:
fixed